### PR TITLE
Date/time clarification..

### PR DESCRIPTION
**Pull request by davebrowning**
...to balance most appropriate use for interoperability as per #957

### CODE REVIEW

File: dcat/index.html [hunk 1]
```diff
@@ -1133,7 +1133,7 @@ <h4>Property: update/modification date</h4>
                 <tbody>
                 <tr><td class="prop">Definition:</td><td>Most recent date on which the item was changed, updated or modified.</td></tr>
                 <tr><td class="prop">Range:</td><td><a href="http://www.w3.org/2000/01/rdf-schema#Literal"><code>rdfs:Literal</code></a>
-                    encoded using the relevant ISO 8601 Date and Time compliant string [[?DATETIME]] and typed using the appropriate XML Schema datatype [[!XMLSCHEMA11-2]]
+                    encoded using the relevant ISO 8601 Date and Time compliant string [[?DATETIME]] and typed using the appropriate XML Schema datatype [[!XMLSCHEMA11-2]]. (xsd:gYear, xsd:gYearMonth, xsd:date, or xsd:dateTime)
```
Comment (dr-shorthair) [inline dcat/index.html]: misplaced full-stop

Comment (davebrowning) [reply dcat/index.html]: Indeed, but was still editing at the time....!

**Review by davebrowning — COMMENTED** [review]
I think this is complete (and debugged!) now...

**Review by dr-shorthair — APPROVED** [review]

**Comment by agbeltran** [comment]
Preview link: https://raw.githack.com/w3c/dxwg/dcat-issue-957/dcat/index.html

**Review by agbeltran — APPROVED** [review]

**Pull request merged**